src/annotator/integrations/test/html-test.js

```diff
@@ -67,11 +74,38 @@ describe('HTMLIntegration', () => {
     assert.equal(integration.describe, fakeHTMLAnchoring.describe);
   });
 
-  describe('#canAnnotate', () => {
-    it('is always true', () => {
+  describe('#getAnnotatableRange', () => {
+    let fakeTrimmedRange;
+
+    beforeEach(() => {
+      fakeTrimmedRange = sinon.stub(FakeTextRange, 'trimmedRange');
```

Comment:
This is not a stubbing strategy I've used before. Does it pass muster or am I overlooking the obvious better way to do this (stubbing a static method on a mocked-imported class)?

Reply:
This looks fine to me. If you mocked a static method on a real import, it would be important to ensure that all such mocks were undone reliably at the end of the test.